cli/src/main/java/com/liferay/blade/cli/command/CreateCommand.java

```diff
@@ -71,6 +72,17 @@ public CreateCommand(BladeCLI bladeCLI) {
 	public void execute() throws Exception {
 		CreateArgs createArgs = getArgs();
 
+		File baseDir = createArgs.getBase();
+
+		if (!isWorkspace(baseDir)) {
```

Comment:
We need to check the `-d ` argument which specifies the directory to create the project in.  That directory may be in a workspace directory, which is a legal command at that point.

Reply:
hey @simonjhy and @Seiphon can you resend with my suggested change? thanks.